utils/scripts/package.json

```diff
@@ -10,6 +10,6 @@
   },
   "scripts": {
     "compile": "npx tsc ./*.ts",
-    "estimateHLGas": "node estimateHLGas.js"
+    "estimateHLGas": "node estimateHLGas.js ethereum polygon"
```

Comment:
Any particular reason about hardcoding the src/dest networks?

I would remove them, and add an explanation in the `README` about how to run that script!

Reply:
Updated the script to not make any hardcoding, and kept it almost the same as original